### DISCUSSION TITLE
Welcome to Power_Theft_Detection Discussions!

### DISCUSSION THREAD

**Discussion by Akash-Tandale001** [Akash-Tandale001/Power_Theft_Detection#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Harshu467** [comment]
@Akash-Tandale001 What was the problems faced during this project??

**Reply by Akash-Tandale001** [reply]
Biggest problem  I faced is that our dataset is not sanitized property i.e dataset contain duplicate value , null value and many more . So we have to process dataset with various parameter and logic.

**Reply by Harshu467** [reply]
Thanks for answer
